scripts/globals/caskets.lua

```diff
@@ -171,7 +171,7 @@ local function dropChance(player)
     --end
 
     if player:hasStatusEffect(tpz.effect.PROWESS_CASKET_RATE) then
```

Comment:
here we'll want to make a few changes.

make dropChance(player) into dropChance(mob) and where it calls dropChance have it pass the mob instead.

Remove the if statement and just do

`prowessCasketsPower = mob:GetHighestCR()`